models/vista3d/scripts/inferer.py

```diff
@@ -25,7 +25,7 @@ class Vista3dInferer(Inferer):
     Args:
         roi_size: the sliding window patch size.
         overlap: sliding window overlap ratio.
-        use_cfp: use class prompt for point head.
+        use_cfp: use class prompt for point head. Deprecated.
```

Comment:
Deprecated here mean this flag will always be False?
I think in model-zoo we can directly remove the deprecated arg since version control, right? @yiheng-wang-nv

Reply:
correct @KumoLiu . Hi @heyufan1995 , can you remove this arg directly?